en/aos-purchase.md

```diff
@@ -49,15 +49,18 @@ If there's a value on the list of unconsumed purchases, proceed with the **Consu
 > To prevent duplicate provision of an item, always check whether the item is being provided in duplicate in the game server.
 >
 
-![consume flow](https://static.toastoven.net/prod_gamebase/DevelopersGuide/purchase_flow_002_2.40.1.png)
+![consume flow](https://static.toastoven.net/prod_gamebase/DevelopersGuide/purchase_flow_002_2.64.0.png)
 
 1. The game client makes a request to the game server to consume the purchase item.
-    * Passes UserID, gamebaseProductId, paymentSeq, and purchaseToken.
+    * Passes UserID, paymentSeq, and purchaseToken.
 2. The game server checks the game DB to see if there is a history of already providing an item with the same paymentSeq.
-    * 2-1. If the item has not been provided yet, call the Gamebase server's Payment Transaction API to verify that the paymentSeq and purchaseToken values are valid.
+    * 2-1. If the item has not been provided yet, call the Gamebase server's Payment Transaction API to verify that the paymentSeq value is matched with the response field and purchaseToken value is valid.
         * [Game > Gamebase > API Guide > Purchase(IAP) > Get Payment Transaction](./api-guide/#get-payment-transaction)
-    * 2-2. If purchaseToken is a normal value, provide the item corresponding to gamebaseProductId to UserID.
-    * 2-3. After providing the item, store UserID, gamebaseProductId, paymentSeq, and purchaseToken in the game DB to prevent duplicate provision or allow for re-provision.
+        * purchaseToken corresponds to **accessToken** in the server API guide document 
+    * 2-2. gamebaseProductId can be found in the response field of the server's Payment Transaction API 
+        * gamebaseProductId also exists in the client's used payment history list, but the value may not be available at the time of reprocessing, so please use the gamebaseProductId value obtained from the server's Payment Transaction API 
+    * 2-3. If the Payment Transaction API call succeeds and the purchaseToken is confirmed to be valid, issues the item corresponding to the gamebaseProductId to the UserID 
+    * 2-4. After issuing the item, saves the UserID, gamebaseProductId, paymentSeq, and purchaseToken in the game DB to prevent duplicate payments or re-payments 
 3. Regardless of whether the item has been provided, the game server completes the item provision by calling the Gamebase server's consume API.
```

Comment:
```suggestions
Regardless of whether the item has been provided, the game server completes the item provision by calling the Gamebase server's consume API so that the unspent payment list is no longer returned.
```

Reply:
@jungparc 댓글로 문장 수정 제안만 해주신 상태인데요, `translaiton-2024-06-25-en` 브랜치에 수정 및 커밋을 부탁드립니다.